variants/NUCLEO_F4x1RE/variant.cpp

```diff
@@ -163,16 +153,15 @@ WEAK void SystemClock_Config(void)
   RCC_ClkInitStruct.APB1CLKDivider = RCC_HCLK_DIV2;
   RCC_ClkInitStruct.APB2CLKDivider = RCC_HCLK_DIV1;
 
+#ifdef ARDUINO_NUCLEO_F401RE
+  if (HAL_RCC_ClockConfig(&RCC_ClkInitStruct, FLASH_LATENCY_2) != HAL_OK) {
+#else /* ARDUINO_NUCLEO_F411RE */
   if (HAL_RCC_ClockConfig(&RCC_ClkInitStruct, FLASH_LATENCY_3) != HAL_OK) {
-    _Error_Handler(__FILE__, __LINE__);
```

Comment:
LSE is lost in this merge ?

Reply:
Yes, this is useless as the RTC will enable the LSE if requested.
https://github.com/stm32duino/Arduino_Core_STM32/blob/1b8664a815b8968bfb23a968b84e5b2fdc422ce4/libraries/SrcWrapper/src/stm32/rtc.c#L114-L122